website/static/bluebook.csl

```diff
@@ -50,6 +50,16 @@
       </else>
     </choose>
   </macro>
+  <macro name="title">
+    <choose>
+      <if variable="keyword">
+        <text variable="title" font-style="italic" suffix=", OSF Preprints "/>
+      </if>
+      <else>
+        <text variable="title" font-style="italic" suffix=", OSF "/>
```

Comment:
Shouldn't this be `suffix=", Open Science Framework"`, as in the diff below?

Reply:
This sort of completes https://openscience.atlassian.net/browse/PLAT-804 in updating project citations to use "OSF" not "Open Science Framework"